### PR TITLE
[Magic WAN] Added Fortinet tutorial

### PR DESCRIPTION
**Pull request by dcpena**
Added Fortinet tutorial to address PCX-4084.

### CODE REVIEW

File: content/magic-wan/tutorials/fortinet.md [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+        set distance 5
+        set priority 10
+        set device "<PHASE1-NAME>"
+    next
+end
+```
+
+### Policy based route
+
+The example below redirects interesting traffic from certain endpoints via the tunnel.
```
Comment (kodster28) [inline content/magic-wan/tutorials/fortinet.md:98]: FWIW, `interesting` feels like the wrong word?

File: content/magic-wan/tutorials/fortinet.md [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+###################################################################################################
+    edit 2 # The policy ID will vary. Check with "sh router static" to determine where to insert the rule
+        set distance 5
+        set priority 10
+        set device "<PHASE1-NAME>"
+    next
+end
+```
+
+### Policy based route
```
Comment (kodster28) [inline content/magic-wan/tutorials/fortinet.md:96]:
```suggestion
### Policy-based route
```

File: content/magic-wan/tutorials/fortinet.md [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+
+The example below redirects interesting traffic from certain endpoints via the tunnel.
+
+```bash
+---
+header: Policy based route
```
Comment (kodster28) [inline content/magic-wan/tutorials/fortinet.md:102]:
```suggestion
header: Policy-based route
```

File: content/magic-wan/tutorials/fortinet.md [hunk 4]
```diff
@@ -0,0 +1,277 @@
...
+        set gateway <Tunnel inside IP - CF side>
+        set output-device "<PHASE1-name>"
+    next
+end
+```
+
+Fortigate is firewall first, and you will need to create Firewall Policies to ensure traffic is allowed between LAN and IPSec.
+
+
+## IPsec - Policy based configuration
```
Comment (kodster28) [inline content/magic-wan/tutorials/fortinet.md:118]:
```suggestion
## IPsec - Policy-based configuration
```

File: content/magic-wan/tutorials/fortinet.md [hunk 5]
```diff
@@ -0,0 +1,277 @@
...
+---
+header: Adjust TCP MSS
+---
+edit "lan" # Change to LAN interface name
+        set tcp-mss 1436
+    next
+end
+```
+
+### Create a policy based route
```
Comment (kodster28) [inline content/magic-wan/tutorials/fortinet.md:260]:
```suggestion
### Create a policy-based route
```

File: content/magic-wan/tutorials/fortinet.md [hunk 6]
```diff
@@ -0,0 +1,277 @@
...
+
+Next, create a Policy Based Route to route desired traffic down the tunnel. The example below tunnels all RFC1918 address spaces.
```
Comment (kodster28) [inline content/magic-wan/tutorials/fortinet.md:262]:
```suggestion
Next, create a Policy-based Route to route desired traffic down the tunnel. The example below tunnels all RFC1918 address spaces.
```

File: content/magic-wan/tutorials/fortinet.md [hunk 7]
```diff
@@ -0,0 +1,277 @@
...
+
+```bash
+---
+header: Policy Based Route
```
Comment (kodster28) [inline content/magic-wan/tutorials/fortinet.md:266]:
```suggestion
header: Policy-based Route
```

**Review by kodster28 — APPROVED** [review]

File: content/magic-wan/tutorials/fortinet.md (hunk 1 above)
Comment (dcpena) [reply content/magic-wan/tutorials/fortinet.md:98]: It's actually a thing! "Interesting traffic is the traffic that is allowed in the encryption domain."

**Pull request merged**